magefiles/build.go

```diff
@@ -298,7 +298,7 @@ func (b Build) buildPackageOperatorImage(imageCacheDir string) error {
 	)
 
 	deployment := &appsv1.Deployment{}
-	err := loadAndUnmarshalIntoObject("deploy-extras/package/hcp/addon-operator-template.yaml", deployment)
+	err := loadAndUnmarshalIntoObject("deploy-extras/package/addon-operator-template.yaml", deployment)
```

Comment:
We don't really have a `package` folder under `deploy-extras`. All the changes in this file can be omitted for now.